recipes/maxima/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "5.39.0" %}
+{% set name = "maxima" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tgz
+  url: https://sourceforge.net/projects/maxima/files/Maxima-source/{{ version }}-source/{{ name }}-{{ version }}.tar.gz
+  sha256: f58ff98013fcaa37d8661b941672c38db58f982f082d6cfcca1f1b5fac7d71b3
+  patches:
+    # Patches from https://git.sagemath.org/sage.git/tree/build/pkgs/maxima/patches
+    - 0001-taylor2-Avoid-blowing-the-stack-when-diff-expand-isn.patch
+    - infodir.patch
+    - maxima.system.patch
+    - matrixexp.patch
+    - undoing_true_false_printing_patch.patch
```

Comment:
I thought you, or someone, might say so.  I don't mind doing it I was just lazy for now since I don't actually know what most of the patches are for.  I took a cursory glance though and most of them seemed to be for actual bug fixes.   A lot of the patches in Sage are like that--either fixes that havent' been released upstream yet, or haven't even been accepted yet due to bikeshedding issues or time availability of upstream to review them.

That said, now that I've got the initial PR up I can take some more time to review these patches more closely.